batch/batch/server/google_storage.py

```diff
@@ -44,7 +44,7 @@ def _wrap_returning_network_call(self, fun):
         return wrapped
 
     def _wrap_nonreturning_network_call(self, fun):
-        fun = self._wrap_nonreturning_network_call(fun)
+        fun = self._wrap_returning_network_call(fun)
```

Comment:
The function names don't match now...

Reply:
They're not supposed to, this was an infinite loop before. `_wrap_returning_network_call` does the normal wrapping stuff. `_wrap_nonreturning_network_call` just drops the returned value.

Reply:
ok.